web/src/services/Selector.service.ts

```diff
@@ -126,6 +126,13 @@ const SelectorService = () => ({
       return Promise.resolve(true);
     return Promise.reject(new Error('Selector already exists'));
   },
+
+  getIsAdvancedSelector(selector: string): boolean {
```

Comment:
How do we know if it's an advanced selector?
1. it has multiple `span` selectors (indicates or operator or ancestor).
2. It has at least one or operator (`],`)